pkg/runner/local_runner_windows.go

```diff
@@ -25,6 +26,24 @@ func NewPlainCommandCreator(sysProcAttr *syscall.SysProcAttr) CommandCreator {
 	return func(ctx context.Context, arguments []string, inputRootDirectory *path.Builder, workingDirectoryParser path.Parser, pathVariable string) (*exec.Cmd, error) {
```

Comment:
If we can't get rid of this CommandCreator implementation, can we at least ensure it's as similar to the one we use on UNIX as possible, in terms of comments, ordering of statements, etc. etc. etc.

Reply:
Yepp!